### DISCUSSION TITLE
Samples using Prism

### DISCUSSION THREAD

**Discussion by e066377** [unoplatform/uno#6563] [Q&A] [answered]
Is it possible to add some sample projects that use Prism.Uno to Uno.Samples?

**Comment by jeromelaban** [comment]
Yes, it's possible! We don't have anything planned for now, but you can use the [`dotnet new unoapp-prism` template](https://platform.uno/docs/articles/get-started-dotnet-new.html), as well as take a look at the [prism repository e2e sample](https://github.com/PrismLibrary/Prism/tree/master/e2e/Uno).

**Comment by davidjohnoliver** [comment, marked as answer]
For a larger reference app that's built using Prism, you can take a look at the Uno Contoso app: https://github.com/unoplatform/Uno.Samples/tree/master/UI/UnoContoso
